README.md

```diff
@@ -3,6 +3,15 @@
 This is open source application used to help parents to communicate and organize
```

Comment:
Hi @sefe
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966](https://user-images.githubusercontent.com/85113567/164451474-70f24aed-2180-4cb6-9226-9ddffb42174f.gif)

## Highlights ⚡ 
✅ All tests are passing 👏 
✅ Professional README file 💯 
✅ Good commit messages 👍 
There is nothing else to say other than... it's time to merge it :ship: 
Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Reply:
It was a nice job and I did approve it, but the screenshots look weird.  PLease fix it.

Reply:
Sorry, I do not understand what is weir and which application is changing every day.
Does this image better explain the main purpose ?
![228525249-89967f4a-e126-4943-9ab9-5494fb7b1f40](https://user-images.githubusercontent.com/1426092/230731951-5dc82241-ea02-4872-ab47-7c14603c1227.png)